op-node/rollup/derive/l1_retrieval_test.go

```diff
@@ -117,6 +133,7 @@ func TestL1RetrievalNextData(t *testing.T) {
 		{
 			name:         "fail to open data",
 			prevBlock:    a,
+			sysCfg:       eth.SystemConfig{BatcherAddr: common.Address{42}},
```

Comment:
Not sure if `Address{42}` is the best test account bc it is a predeploy, the `LegacyMessagePasser` contract